extension/content.css

```diff
@@ -399,21 +399,52 @@ svg.octicon.octicon-star.dashboard-event-icon {
 }
 
 .reaction-summary-item div,
+.reaction-summary-item div a,
 .reaction-summary-item div img {
 	display: inline-block;
 }
 
+.reaction-summary-item div a {
+	transition: margin-left 0.2s;
+	padding-left: 3px;
+}
+
 .reaction-summary-item div img,
 .reaction-summary-item div span {
 	width: 20px;
 	height: 20px;
-	margin-left: 3px;
 	vertical-align: middle;
 }
+
 .reaction-summary-item div img {
+	color: #FFF; /* affects box-shadow */
+	box-shadow: 0 0 0 2px;
 	border-radius: 3px;
 }
 
+.reaction-summary-item.user-has-reacted div img {
+	color: #f2f8fa;
+}
+
+/* overlap reaction avatars when there are 5+ types of reactions */
+.reaction-summary-item:first-child:nth-last-child(n+5) a:nth-child(n+2),
+.reaction-summary-item:first-child:nth-last-child(n+5) ~ * a:nth-child(n+2) {
+	padding-left: 0;
```

Comment:
@bfred-it This creates a design inconsistency with https://github.com/sindresorhus/refined-github/pull/363/files#diff-83ab1d10d74dc75c62ac9cf612cfce51R409

Either we should not add the `padding-left` for all avatars, or we should not remove it in the `:hover` instance.

I'm inclined to remove it from line 409 and make the avatars a little tighter. Seems a little cleaner to me.